synctl/cli.py

```diff
@@ -451,7 +451,7 @@ def pop_size_estimate(self):
             while True:
                 if agent in ["y", "Y"]:
                     worker_nodes = int(self.ask_question("How many worker nodes in your kubernetes cluster?  "))
-                    k8ssensor_pod_count = 3
```

Comment:
k8ssensor pod num by default is 3, and the number is not equal to the work nodes.

Reply:
oh ok but in x-small installation k8sensor replica is 1 and that is why it is giving huge difference in size.